tests/utils/test_skipna.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+from janitor.utils import skipna
+import numpy as np
+
+import pytest
+
+
+@pytest.mark.functions
+def test_skipna():
+    df = pd.DataFrame({"x": ["a", "b", "c", np.nan], "y": [1, 2, 3, np.nan]})
+    func = lambda s: s + "1"
```

Comment:
I like the use of lambda! But to keep the codebase beginner-friendly, can we instead use an internal function definition?